### DISCUSSION TITLE
How to run validation on a button "other than submit" click?

### DISCUSSION THREAD

**Discussion by hazem-alabiad** [react-hook-form/react-hook-form#5841] [Q&A] [answered]
How can I validate the form on a button click, that is other than submit button?

**Comment by bluebill1049** [comment, marked as answer]
`trigger`? https://react-hook-form.com/api/useform/trigger

**Reply by hazem-alabiad** [reply]
How can I trigger all fields?

**Reply by hazem-alabiad** [reply]
Oh I just got it `trigger()`
@bluebill1049 Thanks :)
